composer.json

```diff
@@ -7,18 +7,9 @@
         "email": "nekufa@gmail.com"
     }],
     "require": {
-        "basis-company/framework": "7.0.6",
-        "basis-company/telemetry": "^0.1.11",
-        "ext-decimal": "*",
```

Comment:
@Firetawnyowl @nekufa I'm wondering why this extension was removed. Are decimals no longer supported in the new version (or were they not supported before as well)?

Reply:
Decimals supported, this extension is now installed using "install-php-extensions", not composer

Reply:
If it's supported, why removing the requirement from `composer.json` then?